src/App.tsx

```diff
@@ -1,173 +1,73 @@
-import { Loader } from './components/Loader';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { Person } from './types';
+import { getPeople } from './api';
+import { PeoplePage } from './components/People/PeoplePage';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { Navbar } from './components/Navbar';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  const showLoader = people === null && !isError;
+
+  useEffect(() => {
```

Comment:
you can use people state and this effect at PeoplePage

Reply:
I moved hooks to `PeoplePage`.